src/schema/schema_stat.c

```diff
@@ -19,14 +19,17 @@ __wt_curstat_colgroup_init(WT_SESSION_IMPL *session,
 	WT_COLGROUP *colgroup;
 	WT_DECL_ITEM(buf);
 	WT_DECL_RET;
+	WT_TABLE *table;
 
-	WT_RET(__wt_schema_get_colgroup(session, uri, false, NULL, &colgroup));
+	WT_RET(__wt_schema_get_colgroup(
+	    session, uri, false, &table, &colgroup));
 
 	WT_RET(__wt_scr_alloc(session, 0, &buf));
 	WT_ERR(__wt_buf_fmt(session, buf, "statistics:%s", colgroup->source));
 	ret = __wt_curstat_init(session, buf->data, NULL, cfg, cst);
 
 err:	__wt_scr_free(session, &buf);
+	WT_TRET(__wt_schema_release_table(session, table));
```

Comment:
@agorrod, check out this change.  It is necessary because otherwise the `table` is released inside `__wt_schema_get_colgroup`, and releasing can now cause the table handle to be destroyed.

I wonder whether this could also be causing the crash we have seen reported when opening statistics cursors?

Reply:
That does seem like a possibility. Should I open a ticket to make a similar change in develop?